custom-metrics-autoscaling/direct-to-sd/Makefile

```diff
@@ -0,0 +1,16 @@
+TAG = v0.1.0
+PREFIX = gcr.io/google-containers
+
+build: sd_dummy_exporter
+
+sd_dummy_exporter: sd_dummy_exporter.go
+	go build -a -o sd_dummy_exporter sd_dummy_exporter.go
```

Comment:
Do you expect 99% of the users of Custom Metrics to modify this image?

If the answer is no, and if we're offering this image as `gcr.io/google-containers/sd-dummy-exporter:v0.1.0` (we are) please keep them in their original repo.

They’re source code for a solution we offer, not a GKE tutorial. So they shouldn't be in samples repo.

Reply:
As discussed offline, it's going to be a part of a GKE tutorial. LMK if I should still remove it.

Reply:
Let's keep here for now. I believe you said this image+config is different than the existing repo and customized to be a sample.

Also we should push this to `gcr.io/google-samples` where people like myself have access and it's more relaxed in terms of security requirements.

Reply:
Manifests now point to (not yet existing) images in that registry